### DISCUSSION TITLE
通过堡垒机登录选中跳板机，然后通过跳板机进入到目标机器

### DISCUSSION THREAD

**Discussion by goeoeo** [electerm/electerm#3030] [General]
场景
1.登陆堡垒机a
2.输入跳板机b
3.通过跳板机b 进入目标机器 c
4.目标机器c,再到指定目录

目前通过设置登陆脚本，能完成1.2 
希望再增加一个登陆脚本（可以设定时间间隔），来实现3，4。
或者现有的功能能实现这4步吗

**Comment by goeoeo** [comment]
或者实现配置连接后自动执行快速命令，应该也能实现

**Comment by zxdong262** [comment]
当前的连接跳跃功能可以做到吗？有没有试过？ https://www.bilibili.com/video/BV1mh4y187yQ/

**Reply by goeoeo** [reply]
试过不行，因为登陆堡垒机的时候要选一个机器（环境的一个入口fb），通过这个才能进内网的其它机器

连接跳跃是不是相当于直接从堡垒机，跳到内网机器，没通过入口节点fb, 所以不通

**Reply by zxdong262** [reply]
好的，争取下个版本加入

**Reply by Aliang-code** [reply]
> 好的，争取下个版本加入

请问这个功能实现了吗，目前用连接跳跃写死堡垒机某个跳板机，依然提示Permission denied, please try again.
